elements/rh-site-status/rh-site-status.ts

```diff
@@ -31,108 +81,167 @@ const statusIconsMap: Record<string, TemplateResult> = {
+    if (this.#component) {
+      return this.#component.description ?? this.#component.status;
+    } else if (this.#status) {
+      const statusText = this.#status.description;
+      return TEXT_MAP[statusText as keyof typeof TEXT_MAP] ?? statusText;
+    } else {
+      return 'Error loading status';
+    }
+  };
 
-  async connectedCallback() {
+  connectedCallback() {
     super.connectedCallback();
-    await this.#getStatus();
+    this.#fetch();
   }
 
   render() {
     const { on = '' } = this;
     return html`
       <div id="container" class="${classMap({ [on]: !!on })}">
-        <a href="https://status.redhat.com/" aria-busy="${this.#isLoading ? 'true' : 'false'}" aria-live="polite">
-          ${this.#isLoading ? html`<rh-spinner size="sm"></rh-spinner><span><slot name="loading-text">Loading</slot></span>` : html`
-            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="17" viewbox="0 0 16 17" fill="none">
-              ${this.#icon}
-            </svg>
-            <span>${this.#text}</span>
-          `}
+        <a href="https://status.redhat.com/"
+           aria-busy="${String(this.#loading) as 'true' | 'false'}"
+           aria-live="polite">${this.#loading ? html`
+          <rh-spinner size="sm"></rh-spinner>
+          <span><slot name="loading-text">Loading</slot></span>` : html`
+          <svg xmlns="http://www.w3.org/2000/svg"
+               width="16"
+               height="17"
+               viewbox="0 0 16 17"
+               fill="none">${this.#icon}</svg>
+          <span>${this.#text}</span>`}
         </a>
       </div>
     `;
   }
 
-  async #getStatus() {
+  async #fetch() {
     try {
-      const data: ApiStatus = await fetch('https://status.redhat.com/index.json', {
-        mode: 'cors',
-        cache: 'no-cache',
-        headers: {
-          Accept: 'application/json',
-        },
-      })
-          .then(response => {
-            if (response.ok) {
-              return response.json();
-            } else {
-              throw new Error(`${response.statusText}`);
-            }
-          });
-      if (!RhSiteStatus.isApiStatus(data)) {
-        throw new Error('Invalid status data');
-      }
-      const statusText = data.status.description;
-      this.#text = textMap[statusText] || statusText;
-      this.#icon = statusIconsMap[statusMap[data.status.indicator]];
-      this.#isLoading = false;
-      this.requestUpdate();
+      const { dataURI, fetchOptions } = RhSiteStatus;
+      const data = await fetch(dataURI, fetchOptions)
+          .then(getSummaryOrThrow);
+      const [component] = data.components
+          .filter(x => x.name === this.#domain)
+          .sort(byISO8601Property('updated_at'));
+      this.#component = component ?? null;
+      this.#status = data.status;
     } catch (error) {
       this.#logger.warn('Error loading site status:', error);
-      this.#text = 'Error loading status';
-      this.#icon = statusIconsMap['danger'];
```

Comment:
Removing these makes the status demos no longer conform to the design spec https://www.figma.com/file/j7Rjo2W6PLloxOSXQOpll0/Red-Hat-Design-System-library?type=design&node-id=2447%3A129&mode=design&t=RL9K2HTQV8BGJwgL-1